### PR TITLE
Update UG for weiming

### PR DESCRIPTION
**Pull request by lawwm**

### CODE REVIEW

File: docs/diagrams/SortSequenceDiagram.puml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+create CommandResult
+SortCommand --> CommandResult : result
+activate CommandResult
+
+CommandResult --> SortCommand
+deactivate CommandResult
+
+SortCommand --> LogicManager
+deactivate SortCommand
+destroy SortCommand
```
Comment (bakano98) [inline docs/diagrams/SortSequenceDiagram.puml:71]: `SortCommand -[hidden]-> LogicManager` to destroy the marker below the end of the activation bar. Looks quite wonky as it is right now

**Review by bakano98 — COMMENTED** [review]
Need to fix the `X` on the activation bar. I have added a comment on how to get that to be done -- just add it right below `deactivate SortCommand`

Otherwise, LGTM!

File: docs/diagrams/SortSequenceDiagram.puml (hunk 1 above)
Comment (lawwm) [reply docs/diagrams/SortSequenceDiagram.puml:73]: Thanks for the solution, I've updated the diagram.

**Review by fantablack — COMMENTED** [review]
The write up and sequence diagram are easy to understand, reading it as a fellow developer. The X placement is also better now. LGTM

**Review by Btaykb — COMMENTED** [review]
LGTM

**Pull request merged**